### DISCUSSION TITLE
PyBoard hardware availability

### DISCUSSION THREAD

**Discussion by jirimacku** [micropython/micropython#14063] [General]
Hi,
is there any update on the availability of the pyboard hardware?
Thanks

**Comment by jirimacku** [comment]
Pyboard v1.1 started to be available last week. (https://store.micropython.org/product/PYBv1.1#_). I put the order, seemingly everything went ok and payment was accepted. But the confirmation email mentioned on the webpage never arrived. Also my email to contact@micropython.org remained unanswered. Anybody has similar experience?

**Reply by peterhinch** [reply]
I'm surprised. I last ordered from them in January, they responded with an email and despatched the same day.

**Comment by beetlegigg** [comment]
I've been wanting to hold of one of these but thought they had been discontinued as all was out of stock when I looked over the passed couple of years.   However prompted to look at the store by this post I just put an order in, and, after a short delay I did receive an order confirmation email.  I hope you are by now sorted out with your correspondence with the micropython store and as of now it seems to be working OK.
